zerver/webhooks/slack/tests.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+from zerver.lib.test_classes import WebhookTestCase
+from django.utils.translation import ugettext as _
+from six import text_type
+
+class SlackWebhookTests(WebhookTestCase):
+    STREAM_NAME = 'slack'
+    URL_TEMPLATE = "/api/v1/external/slack?stream={stream}&api_key={api_key}"
+    FIXTURE_DIR_NAME = 'slack'
+
+    def test_slack_channel_to_topic(self):
+        # type: () -> None
+
+        expected_subject = u"channel: general"
+        expected_message = u"**slack_user** has sent the following message:\n `test this\n`"
+        self.send_and_test_stream_message('message_info', expected_subject, expected_message,
+                                          content_type="application/x-www-form-urlencoded")
+
+    def test_slack_channel_to_stream(self):
+        # type: () -> None
+
+        self.STREAM_NAME = 'general'
+        self.url += "&channels_map_to_topics={}".format(0)
```

Comment:
`url = "{}{}".format(self.url, "&channels_map_to_topics=0")`-> format is not needed